shipping/src/main/java/com/instana/robotshop/shipping/Controller.java

```diff
@@ -140,13 +124,15 @@ public List<City> match(@PathVariable String code, @PathVariable String text) {
         if (cities.size() > 10) {
             cities = cities.subList(0, 9);
         }
-
+        long finish = System.currentTimeMillis();
+        long timeElapsed = finish - start;
+        meterRegistry.timer("rt_shipping_get_matchcode").record(Duration.ofMillis(timeElapsed));
```

Comment:
rename metric names as mentioned in the following commit message

https://github.com/drac98/robot-shop/commit/88d2b63ec7732ee60b8ea406fb78ca7e31db757f